packages/react-components/react-tabster/src/focus/createFocusOutlineStyle.ts

```diff
@@ -59,6 +59,11 @@ const getFocusOutlineStyles = (options: FocusOutlineStyleOptions): GriffelStyle
 
   return {
     ...shorthands.borderColor('transparent'),
+    '@media (forced-colors: active)': {
+      '::after': {
+        ...shorthands.borderColor('Highlight'),
+      },
+    },
     '::after': {
```

Comment:
```suggestion
    '::after': {
        '@media (forced-colors: active)': {
           ...shorthands.borderColor('Highlight'),
        },
    },
```

nit: this should also work?